admin/class-metabox.php

```diff
@@ -1210,6 +1224,31 @@ function column_sort_orderby( $vars ) {
 	}
 
 	/**
+	 * Hide the SEO Title, Meta Desc and Focus KW columns if the user hasn't chosen which columns to hide
```

Comment:
The reason it's like that is because the column is actually called `Focus KW` within the WordPress interface.